testing/upload_validation.ts

```diff
@@ -932,7 +934,11 @@ const baseSyncTableSchema = {
   entityName: z.string().optional(),
   defaultAddDynamicColumns: z.boolean().optional(),
   // TODO(patrick): Make identityName non-optional after SDK v1.0.0 is required
-  identityName: z.string().min(1).optional(),
+  identityName: z
+    .string()
+    .min(1)
+    .optional()
+    .refine(val => !val || !SystemColumnNames.includes(val), 'Invalid identityName'),
```

Comment:
Maybe a slightly longer and friendlier error message, like "This property name is reserved for internal use by Coda and can't be used as an identity name, sorry!"

Reply:
Done, thanks!